files/zookeeper.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Zookeeper distributed coordination server
+After=network.target
+Wants=network.target
```

Comment:
Is this right? Both `After`/`Wants` = network.target?

Reply:
Let me double check this. I guess this means, it is dependent on network.target.

Reply:
As per https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Before= 

"It is a common pattern to include a unit name in both the After= and Requires= options, in which case the unit listed will be started before the unit that is configured with these options"